### PR TITLE
refactor: replace `execa` with `execSync` in sample tests

### LINKED ISSUE #354: Refactor to use 'execSync' in tests instead of 'execa'
Everywhere there is an 'execa', we should be using 'execSync'. 
Nodejs-translate provides a good example of what 'execSync' should look like https://github.com/googleapis/nodejs-translate/tree/master/samples/test

### PR DESCRIPTION
**Pull request by jbdoster**
Fixes #354  (it's a good idea to open an issue first for discussion)

- [x] Tests and linter pass
- [? How can I check this?] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: samples/system-test/detect.test.js [hunk 1]
```diff
@@ -17,12 +17,12 @@
...
 const path = require('path');
 const {assert} = require('chai');
-const execa = require('execa');
-
+const cp = require('child_process');
 const cmd = 'node detect.js';
 const cmd_tts = 'node detect-intent-TTS-response.v2.js';
 const cmd_sentiment = 'node detect-intent-sentiment.v2.js';
 const cwd = path.join(__dirname, '..');
+cp.cwd = cwd;
```
Comment (JustinBeckwith) [inline samples/system-test/detect.test.js]: We can just delete lines 24 and 25 here

Comment (JustinBeckwith) [reply samples/system-test/detect.test.js]: Also - we like to add a line like this:

```js
const exec = cmd => execSync(cmd, { encoding: 'utf8' });
```

We use this pretty consistently to make sure the data returned from `execSync` is properly encoded.  We also generally don't do the `pipe` bit below

File: samples/system-test/detect.test.js [hunk 2]
```diff
@@ -33,33 +33,33 @@ const audioFilepathBookARoom = path
...
-    const {stdout} = await execa.shell(`${cmd} event WELCOME`, {cwd});
+    const stdout = cp.execSync(`${cmd} event WELCOME`, {stdio: 'pipe'});
     assert.include(stdout, 'Query: WELCOME');
   });
 
   it('should detect audio query', async () => {
-    const {stdout} = await execa.shell(
+    const stdout = cp.execSync(
       `${cmd} audio ${audioFilepathBookARoom} -r 16000`,
       {cwd}
```
Comment (JustinBeckwith) [inline samples/system-test/detect.test.js]: We can drop the `{cwd}` where you see it in most parts

File: samples/system-test/detect.test.js [hunk 3]
```diff
@@ -70,10 +70,10 @@ describe('basic detection', () => {
...
   it('should detect sentiment with intent', async () => {
-    const {stdout} = await execa.shell(
+    const stdout = cp.execSync(
       `${cmd_sentiment} ${projectId} 'SESSION_ID' '${testQuery}' 'en-US'`,
       {cwd}
     );
     assert.include(stdout, 'Detected sentiment');
   });
-});
+});
```
Comment (JustinBeckwith) [inline samples/system-test/detect.test.js]: Let's add these newlines at the end of the files back to keep the angry red circle away :)

Comment (jbdoster) [reply samples/system-test/detect.test.js]: Got it, thank you so much! I will remember that. I applied the changes, please let me know if anything else needs to be touched up and I will square it away :-)

File: samples/system-test/detect.test.js [hunk 4]
```diff
@@ -31,48 +29,43 @@ const audioFilepathBookARoom = path
   .join(__dirname, '../resources/book_a_room.wav')
   .replace(/(\s+)/g, '\\$1');
 
+const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+
 describe('basic detection', () => {
   it('should detect text queries', async () => {
-    const {stdout} = await execa.shell(`${cmd} text -q "hello"`, {cwd});
+    const stdout = await exec(`${cmd} text -q "hello"`);
```
Comment (JustinBeckwith) [inline samples/system-test/detect.test.js]: Since this method is sync, you don't need to await it!

Comment (jbdoster) [reply samples/system-test/detect.test.js]: Oops! I removed all the unnecessary awaits and pushed a new commit, thank you!

**Comment by JustinBeckwith** [comment]
Apologies for the delay!  Could we trouble you to run `npm run fix`, and push and another commit to fix the linting errors?

**Comment by jbdoster** [comment]
> Apologies for the delay! Could we trouble you to run `npm run fix`, and push and another commit to fix the linting errors?

Hey Justin,

No worries! I ran npm run fix and I am receiving the error  
`error  "pb-util" "chai" "dialogflow" is not found  node/no-missing-require`

I'll be honest, I did npm install and globally saved those packages but I am not sure how to fix that :-( 
I will continue to work out the issue, sorry!

**Pull request merged**